os/linux/weston/plugin/compositor-iahwc.c

```diff
@@ -1672,8 +1671,9 @@ static void session_notify(struct wl_listener *listener, void *data) {
   }
 }
 
-static void planes_binding(struct weston_keyboard *keyboard, uint32_t time,
-                           uint32_t key, void *data) {
+static void
+planes_binding(struct weston_keyboard *keyboard, const struct timespec* time,
```

Comment:
can u separate indentation fixes to a separate patch

Reply:
Perhaps fix any other indentation issues same time

Reply:
ok, sure.